OpenImis.ModulesV2/Helpers/Extensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Xml.Serialization;
+
+namespace OpenImis.ModulesV2.Helpers
+{
+    public static class Extensions
+    {
+        public static string Serialize<T>(this T value)
```

Comment:
Can we be more explicit like XMLSerialize as to not be mistaken with the default Serializer?

Reply:
Ok, I will change to `XMLSerialize`

Reply:
Done